aws-android-sdk-kinesisvideo/src/main/java/com/amazonaws/mobileconnectors/kinesisvideo/service/KinesisVideoAndroidServiceClient.java

```diff
@@ -192,7 +192,8 @@ public String createStream(@NonNull final String streamName,
                 .withDeviceName(deviceName)
                 .withMediaType(contentType)
                 .withKmsKeyId(StringUtils.isBlank(kmsKeyId) ? null : kmsKeyId)
-                .withDataRetentionInHours((int) retentionPeriodInHours);
+                .withDataRetentionInHours((int) retentionPeriodInHours)
+                .withTags(null);
```

Comment:
Will there be a case where tags can be passed? Can the developer override this?

Reply:
No. It is not expected to be passed on creation now. Passing the tags on creation is complex change also related to SDK state machine change. This feature might be support later.

Reply:
Can you add a comment in the code to explain why tags are passed as null?

Reply:
Also, please add a statement in CHANGELOG.md file as a bug fix.

Reply:
Updated.